features/dd-trace-core/lint.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+    <issue id="all">
+        <ignore path="*"/>
```

Comment:
> [!WARNING]
> Is this really a good idea to ignore all lints? I get why we might want this while making the module but we should not commit this

Reply:
Since we're pulling the code into our repo, we become responsible for it, and must be sure that it's going to work and meet our reliability criterias. 

Ignoring **all** issues is a disaster waiting to happen. I agree we might not want to fix all the issues right now, but we need to make sure we didn't miss anything important. 

I agree that it will be tedious but let's have more granularity on which issues we suppress (with a JIRA ticket to fix it later) and which need to be fixed right now.

Reply:
I agree, let's see what we can address and what we cannot.